code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -150,12 +150,21 @@
 	siemens_coefficient = -1
 	var/tesla_power = 25000
 	var/tesla_range = 20
-	var/tesla_boom = FALSE
-	var/tesla_stun = FALSE
+	var/tesla_flags = TESLA_MOB_DAMAGE | TESLA_OBJ_DAMAGE
+
+/obj/item/clothing/suit/armor/reactive/tesla/dropped(mob/user)
+	..()
+	if(istype(user))
+		user.flags_1 &= ~TESLA_IGNORE_1
+
+/obj/item/clothing/suit/armor/reactive/tesla/equipped(mob/user, slot)
```

Comment:
going to point this out so no one screeches. it prevents tesla armor nerds from getting targeted by tesla bolts. was in the original pr but people might have different opinions now that tesla armor is a thing.